### DISCUSSION TITLE
Show packages' peer dependencies on npmjs.com

### DISCUSSION THREAD

**Discussion by val1984** [npm/feedback#693] [Website] [answered]
Since npm 7.0.0, peer dependencies are installed by default (unless you use the `legacy-peer-deps` option) but these are not listed on npmjs.com website.
Considering these are installed, I believe that these should be shown in the dependencies tab of each package, in a "Peer dependencies" section, preferably with their version requirement. Here's a quick'n'dirty mockup:
![image](https://user-images.githubusercontent.com/3774962/169766863-30ffc6e9-15c1-4def-999c-f00b74a49579.png)
Any thoughts/feedback on this?
Thank you!

**Comment by monishcm** [comment, marked as answer]
We will investigate this and add to our backlog. Thanks for your feedback.
